container-conf/build.sh

```diff
@@ -8,3 +8,13 @@ build_as_root() {
     build_yum config-manager --set-enabled google-chrome
     build_yum install google-chrome-stable
 }
+
+build_as_run_user() {
+    umask 022
+    cd ~
```

Comment:
`cd ~` is same as `cd`. However, neither should be done. Downloads should happen in the [build_dir](https://github.com/radiasoft/containers/blob/a86906589709cabbe6bfffb62497820dfb94ac7d/bin/build.sh#L299) which is the current directory when the build is run.

Reply:
91034b60c74e645b2b6c07a317c7888a4fa06831